src/components/MarkdownTable.tsx

```diff
@@ -14,6 +14,9 @@ const MarkdownTable: React.FC<IProps> = ({ children }) => (
           borderColor: "border",
           whiteSpace: "nowrap",
         },
+        ".chakra-link .visually-hidden": {
+          width: "0px",
+        },
```

Comment:
I think this issue is related to the absolute positioning of the hidden div and the `overflowX` of the table. For some reason, the absolute positioned div is overflowing the container.

In short, I think we can add a `position: relative` on the table's container and it will fix it without the need of adding a new class or style on the Link itself.

Reply:
Fixed :)